hole/mahjong.go

```diff
@@ -200,7 +200,7 @@ func isHandValid(tileCounts map[rune]int) bool {
 func mahjong() []Run {
 	runs := make([]Run, 10)
 
-	for i := range runs {
+	for i := 0; i < len(runs)-1; i++ {
```

Comment:
What's this change for? It seems unrelated, unnecessary, and less idiomatic.

Reply:
Rather than doing 10 randomly generated runs, it now does 9 random runs and 1 run with the static test cases, so we stop one run before the end of the array when generating the random test cases so that we can later add the run with the static test cases.
I'm not too familiar with golang, so there's probably a cleaner way of expressing this.

Reply:
Ah okay, that makes sense, my bad.